### DISCUSSION TITLE
We have updated our Italian VisiData guide

### DISCUSSION THREAD

**Discussion by aborruso** [saulpw/visidata#1195] [Show and tell]
Hi,
I have been following and loving VisiData almost from its start, and after seeing the [great Jeremy's guide](https://jsvine.github.io/intro-to-visidata/), I wanted to create a guide in Italian. This was a long time ago.

This small guide helped to make this great product known here in Italy and the more time passed, the more it was necessary to create a better version.
That time has come 💜: https://ondata.github.io/guidaVisiData/

At the moment there is no great new content, but there is a new structure and a new engine (the great Material for MkDocs), which make it very readable and usable. On the [note page](https://ondata.github.io/guidaVisiData/note/), I make some thanks, which I also want to write here:

- to @saulpw , for having invented VisiData and for being a brilliant and generous person;
- to @anjakefala , for being always present and attentive in the management of project issues and for her contribution to the development of VisiData;
- to @jsvine , for writing the most beautiful VisiData guide (for me), for being a constant source of inspiration.

I should add so many more thanks, but I make one the fantastic community of this product.

**Comment by anjakefala** [comment]
<3333333333333333333333

I really appreciate you doing the important work of translating and sharing VisiData in a non-English context. You have been a very lovely, friendly icon-face through the years. <(^___^)>

Mille grazie, Andrea!

**Comment by jsvine** [comment]
Bravissimo, Andrea! E grazie per le gentili parole!
